### ISSUE TITLE
Need to increase number of verification tests

### ISSUE THREAD

**Issue by andrewlee94** [IDAES/idaes-pse#172]
We need to include more model verification tests across a range of use cases. Currently testing of dynamic cases is rather spares.

**Comment by eslickj** [comment]
Yeah.  It would be really nice if we had a source of models with known results.  I was hoping to find some simple test set for the PID controller testing, but haven't yet.

**Comment by Robbybp** [comment]
@andrewlee94 What do you mean by model verification? Testing that use cases work from a software/tractability perspective, or that they give us the results we expect (possibly by comparing with external results)?

**Comment by andrewlee94** [comment]
@Robbybp In reality, both. We need to test more case studies to look for any bugs or edge-cases in the code and also to verify the model results against known sources.
